src/objects/viewport.ts

```diff
@@ -0,0 +1,28 @@
+import { Viewport } from 'pixi-viewport'
+import { Sprite, Texture } from 'pixi.js'
+import { application } from '@/objects/application'
+
+export let viewport: Viewport
+
+export function createViewport(): void {
+  viewport = new Viewport({
+    events: application.renderer.events,
+    passiveWheel: false,
+  })
+
+  viewport
+    .drag()
+    .pinch()
+    .wheel()
+    .decelerate({
+      friction: 0.9,
```

Comment:
For my own knowledge, is there some significance to the friction value?

Reply:
This was a value carried over from the previous version. Without some friction if you release while moving everything just flies off screen.

Reply:
it occurs to me now that my comment on the PR is probably related to this value, since this is passed to a function called "decelerate" 🤦🏻